dkist/dataset/tiled_dataset.py

```diff
@@ -167,7 +170,26 @@ def _get_axislabels(ax):
                 ylabel = coord.get_axislabel() or coord._get_default_axislabel()
         return (xlabel, ylabel)
 
-    def plot(self, slice_index, share_zscale=False, fig=None, **kwargs):
+    @staticmethod
+    def _ensure_wcs_ordered_axis_lims(wcs: HighLevelWCSWrapper, ax: Axes) -> None:
```

Comment:
I am generally uneasy about this whole thing. `WCSAxes` is weird and I worry this is poking a bear.